fastlane/Fastfile

```diff
@@ -664,6 +673,41 @@ import "./ScreenshotFastfile"
     )
   end
 
+  #####################################################################################
+  # upload_gutenberg_sourcemaps
+  # -----------------------------------------------------------------------------------
+  # This lane uploads the Gutenberg source maps
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane upload_gutenberg_sourcemaps [internal:<internal>]
+  #
+  # Example:
+  # bundle exec fastlane upload_gutenberg_sourcemaps internal:true
+  #####################################################################################
+  lane :upload_gutenberg_sourcemaps do | options |
+    build_version = ios_get_build_version(internal: options[:internal])
```

Comment:
The build version is different depending on the `internal` option, here is an example:
- `internal: true`: `17.6.0.20210614` ([reference](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/config/Version.internal.xcconfig))
- `internal: false`: `17.6.0.0` ([reference](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/config/Version.public.xcconfig))